src/main/java/JohnChatter.java

```diff
@@ -0,0 +1,84 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class JohnChatter {
+    public static void main(String[] args) throws JohnChatterException {
+        System.out.println("it is i, john chatter");
+
+        ArrayList<Task> items = new ArrayList<>();
+        String input;
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            // Wait for user inputs. Breaks out and returns when the command "bye" is given
+            input = scanner.nextLine();
+            String[] splitInputAroundSpace = input.split(" ");
+
+            try {
+                if (input.equals("bye")) {
+                    System.out.println("goodbye!");
+                    return;
+                } else if (input.equals("list")) {
+                    for (int i = 1; i <= items.size(); i++) {
+                        Task item = items.get(i -1);
```

Comment:
minor nit: there should be a space after `-`

```suggestion
                        Task item = items.get(i - 1);
```